data/data/bootstrap/ovirt/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
+
+approvers:
```

Comment:
I think you need a section for reviewers too.. and similarly in other OWNERS files.